package-manager-configs/README.md

```diff
@@ -3,8 +3,8 @@ to the associated repository (staging or production).
 The deployment of these files is a one-time operation.
 These files make it easier for users to set up their local machine to use our installer repositories.
 
-Staging: The [nx-staging](nx-staging) files have been deployed to [nx-staging.sonatype.com](https://nx-staging.sonatype.com)
-in the `community-hosted` raw repository.
+Staging: The [nx-staging](nx-staging) files have been deployed to [repo.sonatype.com](https://repo.sonatype.com)
```

Comment:
I don't think these links should change because they refer to the old `nx-staging`. Similarly, the repo name (`community-hosted`) is the same on the staging server as the production server.

Or is a change a-foot to the nx-staging repo names? If so, there's lot's of other changes that will need to happen (e.g. CI config, etc).